datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -343,6 +346,12 @@ public FlowIdentifier rotateAutoTlsCertificatesByCrn(@ResourceCrn @TenantAwarePa
         return certRotationService.rotateAutoTlsCertificates(sdxCluster, rotateCertificateRequest);
     }
 
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    public void changeImageCatalog(@ResourceName @NotEmpty String name, @Valid @NotNull  SdxChangeImageCatalogRequest changeImageCatalogRequest) {
+        // FIXME: Implement this!
```

Comment:
API first approach so we can work in parallel on the CLI.
It is going to be implemented as a part of CB-14309